components/MetroBarChart.vue

```diff
@@ -22,48 +22,52 @@
 }
 </style>
 
-<script>
+<script lang="ts">
+import Vue from 'vue'
 import DataView from '@/components/DataView.vue'
 
-export default {
+export type DataType = {
+  value: string
+}
+
+export default Vue.extend({
   components: { DataView },
   props: {
     title: {
       type: String,
-      required: false,
       default: ''
     },
     chartData: {
       type: Object,
-      required: false,
       default: () => {}
     },
     chartOption: {
-      type: Object,
-      required: false,
+      type: Object as any,
       default: () => {}
     },
     date: {
       type: String,
-      required: true,
-      default: ''
+      required: true
     }
   },
   computed: {
     displayData() {
-      const colors = ['#a6e29f', '#63c765', '#008b41']
-      return {
-        labels: this.chartData.datasets.map(d => d.label),
-        datasets: this.chartData.labels.map((label, i) => {
+      const colors: Array<string> = ['#a6e29f', '#63c765', '#008b41']
```

Comment:
```suggestion
      const colors: string[] = ['#a6e29f', '#63c765', '#008b41']
```

`Array<T>` は古めの書き方で、現在は `T[]` が推奨されているので、こちらでどうでしょう？

Reply:
そうなんですね、ありがとうございます。 `T[]` に修正しました。 
ab31994